src/utils/Utils.ts

```diff
@@ -35,14 +39,34 @@ export default class Utils {
     });
   }
 
-  public static resourceInstancesToList(resourceInstances: ResourceInstance[]): ListEntry[] {
+  public static resourceInstancesToList(resourceInstances: ResourceInstance[], onClick?: clickHandler): ListEntry[] {
     // create list for specific type
     return resourceInstances.map((resourceInstance) => {
       return {
         id: `${resourceInstance.id}`,
         // first value needs to be the identifying value
         firstValue: `${resourceInstance.id}`,
         secondValue: `Resource Type: ${resourceInstance.resourceType.name}`,
+        link: {
+          link: onClick ? '' : `/resources/${resourceInstance.id}`,
+          onClick: onClick ? () => { onClick(resourceInstance.id || ''); } : undefined,
+        },
+      };
+    });
+  }
+
+  public static resourceInstanceAttributesToList(
+    attributes: ResourceInstanceAttribute[],
+    onClick?: clickHandler,
```

Comment:
because you can't click the attributes because there is no link set, we can remove this line, right?